src/unitxt/operators.py

```diff
@@ -376,6 +376,67 @@ def process_value(self, value: Any) -> Any:
         return new_value
 
 
+class AugmentSuffix(Augmentor):
+    """
+    Augments the input by appending to it a randonly selected (typically, whitespace) pattern.
+    Any existing trailig whitespace (if any), of the input is first discarded.
+    Then, a pattern is randomly selected from a given list of (pattern, weight) pairs.
+    Finally, the selected pattern is appended to the (potentially trimmed at its end) input.
+
+    Args:
+     suffixes : the potential white-space patterns to select from.
+        the dictionary version allowes to specify relative weights of the different patterns.
+
+    Examples:
+        to append a '\n' or a '\t' to the end of the input, employ
+        AugmentSuffix(augment_model_input=True, suffixes=['\n','\t'])
+
```

Comment:
Add example with dictionary.

Reply:
Done. and also added the new flag, per your good idea.